src/jira/client/index.ts

```diff
@@ -153,13 +153,22 @@ async function getJiraClient(
 							}
 						}
 					),
-				delete: (gitHubInstallationId: string | number) =>
-					instance.delete(`/rest/devinfo/0.10/bulkByProperties`,
-						{
-							params: {
-								installationId: gitHubInstallationId
-							}
-						})
+				delete: async (gitHubInstallationId: string | number) =>
```

Comment:
This `delete` function is under `devinfo`, so it's technically not the right place to add calls to the `builds` and `deployments` APIs, but I didn't want to restructure the client as it's not typed throughout the whole codebase...

Reply:
yeah, this file need restructuring anyways.  Need a jira client equivalent of our github client.